.github/workflows/test.yml

```diff
@@ -50,23 +50,25 @@ jobs:
         cache: 'pip'
         cache-dependency-path: 'poetry.lock'
 
-    - name: Install Poetry
+    - name: Upgrade pip
       env:
         PIP_CONSTRAINT: .github/workflows/resources/constraints.txt
       run: |
-        pipx install poetry
-        poetry --version
+        pip install pip
+        pip --version
 
-    - name: Upgrade pip
+    - name: Install pipx
       env:
         PIP_CONSTRAINT: .github/workflows/resources/constraints.txt
       run: |
-        pip install pip
-        pip --version
+        python -m pip install pipx
+        pipx --version
```

Comment:
AFAICT GitHub runners come with `pipx` out of the box. Would be good to confirm, though

Reply:
It does come with `pipx` out of the box documented [here](https://github.com/actions/virtual-environments/blob/main/images/win/Windows2022-Readme.md#:~:text=22.2%20(python%203.9)-,Pipx%201.1.0,-RubyGems%203.2.33) the issue is that this causes our issue here https://github.com/meltano/meltano/issues/6479 of Windows running the wrong version of Python while doing testing as the `pipx` out of the box is installed with Python 3.9. There's others way to handle this but I think being explicit like this is probably the easiest to understand 🤷

Reply:
@visch per my comment on https://github.com/meltano/meltano/issues/6479#issuecomment-1201855709  I don't think we need to focus on that since #6395 landed, and we should probably focus on the actual Windows failures that have been surfaced. Wdyt?

Reply:
> > Going to focus first on why the https://github.com/meltano/meltano/runs/7509637290?check_suite_focus=true "Install Dependencies" task isn't reporting a failure to github actions.
> 
> @visch that's an intriguing issue with GitHub actions in general, but may not be too relevant anymore at least for our Windows case, since #6395 got merged. I'd be in favor of changing the title of this issue to reference the actual Windows failures, or closing it and create new and separate issues for the failures on Python 3.7 and 3.10. The former I think is the trickier one since it seems to relate to `asyncio`.

+1

Posted the [3.7 issue here](https://github.com/meltano/meltano/issues/6544) , [3.10 issue here](https://github.com/meltano/meltano/issues/6545)

I get what you're saying now, `nox` works properly on the different versions of python on Windows. 

I was really focused on
![image](https://user-images.githubusercontent.com/8680264/182265734-f21395b8-2854-46c1-9c7a-fa8a9fe6e65a.png)

I thought that's what all of this was about, but you already made something else work here. Would have been smarter for me to open a seperate issue I think? I was just extremely confused with that picture failing.

We could update the https://github.com/meltano/meltano/issues/6479 issue to be [Bug]: Window s fails silently on Configure Poetry step . 

Maybe that makes the most sense from here (I can't update the issue name :D)

Reply:
@visch awesome, thanks!